src/store/index.ts

```diff
@@ -1,29 +1,11 @@
 // Libraries
-import { createPinia, PiniaVuePlugin } from 'pinia'
-import Vue from 'vue'
-import Vuex from 'vuex'
-
-// Store modules
-import * as State from './state'
-
-/**
- * Configures and returns Vuex Store.
- */
-export function getVuexStore () {
```

Comment:
No more Vuex 😄 Can remove since also it's usage has been removed from sbc-common-components.